bitswap/message/message.go

```diff
@@ -164,17 +141,23 @@ func newMsg(full bool) *impl {
 
 // Clone the message fields
 func (m *impl) Clone() BitSwapMessage {
```

Comment:
This could be simplified to

```go
func (m *impl) Clone() BitSwapMessage {
	return &impl{
		full:           m.full,
		wantlist:       maps.Clone(m.wantlist),
		blocks:         maps.Clone(m.blocks),
		blockPresences: maps.Clone(m.blockPresences),
		pendingBytes:   m.pendingBytes,
	}
}
```

Reply:
Done